src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('li')];
+
+  list.sort((a, b) => {
+    const item1 = parseInt(a.getAttribute('data-salary').replace(/[\D]/g, ''));
+    const item2 = parseInt(b.getAttribute('data-salary').replace(/[\D]/g, ''));
+
+    return item2 - item1;
+  });
+
+  list.forEach((item) => {
+    document.querySelector('ul').appendChild(item);
+  });
+}
+
+function getEmployees() {
+  const list = [...document.querySelectorAll('ul')];
+
+  return list.map((item) => ({
+    name: item.textContent(),
+    position: item.dataset.position,
+    salary: item.dataset.salary,
+    age: parseInt(item.dataset.salary),
```

Comment:
You are using `item.textContent()` and `item.dataset` to get the values. However, `textContent` is a property, not a method. You should use `item.textContent` instead. Also, `dataset` properties return string values. You should convert `salary` and `age` to numbers.